backend/x11/output.c

```diff
@@ -217,22 +187,61 @@ struct wlr_output *wlr_x11_output_create(struct wlr_backend *backend) {
 	wlr_signal_emit_safe(&x11->backend.events.new_input, &output->pointer_dev);
 	wlr_signal_emit_safe(&x11->backend.events.new_input, &output->touch_dev);
 
+	// Start the rendering loop
+	wlr_output_send_frame(&output->wlr_output);
```

Comment:
I don't think we need to do this. The compositor will damage the output and call `wlr_output_schedule_frame`.

Reply:
Done